src/web/components/Home/RotateAPIKeysCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Link } from 'react-router-dom';
+
+import { ApiKeyDTO } from '../../../api/services/adminServiceHelpers';
+import {
+  deduplicateStrings,
+  formatStringsWithSeparator,
+  formatUnixDate,
+} from '../../utils/textHelpers';
+
+import './RotateAPIKeysCard.scss';
+
+type RotateApiKeysCardProps = Readonly<{
+  apiKeysToRotate: ApiKeyDTO[];
+}>;
+
+function RotateApiKeysCard({ apiKeysToRotate }: RotateApiKeysCardProps) {
+  const getDatesCreated = () => {
+    const datesCreated = apiKeysToRotate.map((apiKey) => formatUnixDate(apiKey.created).toString());
+    const datesCreatedString = formatStringsWithSeparator(deduplicateStrings(datesCreated));
+    return datesCreatedString;
+  };
+
+  return (
+    <div className='rotate-api-keys-card'>
+      <FontAwesomeIcon icon='triangle-exclamation' className='warning-button' />
+      <h2>Rotate Your API Key{apiKeysToRotate.length > 1 && 's'}</h2>
+      <div className='rotate-api-keys-text'>
+        <span>
+          {`Your API key${
+            apiKeysToRotate.length > 1 ? 's were' : ' was'
+          } created on ${getDatesCreated()}. We recommend you rotate your API key${
+            apiKeysToRotate.length > 1 ? 's' : ''
+          } every year.`}{' '}
+        </span>
+      </div>
+
+      <Link to='/dashboard/apiKeys'>
```

Comment:
I'm wondering if we should actually import `ApiKeyManagementRoute` then use `<Link to={ApiKeyManagementRoute.path}>`. That's the approach I used here: https://github.com/IABTechLab/uid2-self-serve-portal/pull/443/files. Happy to discuss if that seems wrong though

Reply:
What is the difference between the two - what makes the route a better solution?  I'm open to either, we should just have consistency because I took this example from `SharingPermissionsCard.tsx` so if we change this pattern, I'll make sure to change in both places

Reply:
Just to avoid having magic strings lying around. In the unlikely event that we change the path for the `ApiKeyManagementRoute`, we would have to change it in several places if they are hardcoded. Whereas if we used `ApiKeyManagementRoute.path` we'd only have to update it at the source.

I can't think of any downside...

Reply:
That makes sense - changed!